omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,410 @@
+Administrators with restricted privileges
+=========================================
+
+
+Summary
+-------
+
+OMERO allows you to create administrators with a subset of the full
+administrator privileges. This is a way to cater for the need for
+more powerful users acting on behalf of all other OMERO users,
+with no group allegiance but with access to all groups and data of
+all users in OMERO. This should be achieved without creating new
+full administrators in OMERO.
+In the real world, these administrators with restricted privileges
+(restricted admins) will typically be imaging
+facility managers, image analysts, or anybody who needs to organize
+users and data of others' in OMERO.
+
+Full administrators in OMERO can create new administrators with
+restricted privileges using the OMERO.web
+interface (hook to Help with Webadmin graphic screenshots).
```

Comment:
I would use `:help:`<sharing-data.html#admin>` as the reference here - we can add an extra step in between the current steps 2 and 3 for creating these new admins and using an existing anchor means the link won't be broken until the live help is updated at the release